tools/integration_tests/util/operations/file_operations.go

```diff
@@ -437,3 +439,105 @@ func ClearCacheControlOnGcsObject(gcsObjPath string) error {
 	_, err := ExecuteGsutilCommandf("setmeta -h \"Cache-Control:\" gs://%s ", gcsObjPath)
 	return err
 }
+
+func CreateFile(filePath string, filePerms os.FileMode, t *testing.T) (f *os.File) {
```

Comment:
Should we capture error here or as a return statement?

Reply:
Don't see a need because the method itself will log the error and FAIL incase there is any error.